documentation/docs/05-modules.md

```diff
@@ -22,7 +22,7 @@ import { amp, browser, dev, mode, prerendering } from '$app/env';
 import { goto, invalidate, prefetch, prefetchRoutes } from '$app/navigation';
 ```
 
-- `goto(href, { replaceState, noscroll, keepfocus, state })` returns a `Promise` that resolves when SvelteKit navigates (or fails to navigate, in which case the promise rejects) to the specified `href`. The second argument is optional:
+- `goto(href, { replaceState, noscroll, keepfocus, state })` returns a `Promise` that resolves when SvelteKit navigates (or fails to navigate, in which case the promise rejects) to the specified `href`. Causes the `load` function of the `href` to be called. The second argument is optional:
```

Comment:
this change seems unnecessary to me. can you explain why you thought it wouldn't be called?

Reply:
yes this seems unnecessary, i will remove it.